src/styles/main.scss

```
@@ -0,0 +1,11 @@
+@import "utils/reset.scss";
+
+@import "blocks/clock.scss";
+
+.main {
```

And move to `page.scss`

```suggestion
.page {
```